### PR TITLE
Tatiana meisner

### PR DESCRIPTION
**Pull request by meyta2020**

### CODE REVIEW

**Review by Elena-Sergienko — APPROVED** [review]

File: test/specs/roles/new.spec.js [hunk 1]
```diff
@@ -13,27 +13,27 @@ describe('ROLE NEW', () => {
...
+  //   expect(TermsServicePage.headerService).toHaveText(expectedValue.texts.headerTermsService);
+  // });
 
-  it('TC- 6.009 Validate that Role New can login and logout', function () {
-    expect(ProfilePage.userIcon).toBeDisplayed();
-    HomePage.userDropdownMenu.dropdownMenu.click();
-    HomePage.userDropdownMenu.logout.click();
-    expect(LoginPage.loginPageHeader).toHaveText(expectedValue.texts.loginPageHeader);
-  });
+  // it('TC- 6.009 Validate that Role New can login and logout', function () {
```
Comment (ayessetova) [inline test/specs/roles/new.spec.js:33]: did you disabled the tests because they don't pass?

File: test/specs/login.spec.js [hunk 2]
```diff
@@ -50,36 +51,66 @@ describe('Login page', () => {
...
 
-  describe('Login role New', () => {
-    it('TC -2.061 Valid email and password', () => {
+    it('TC-2.021 User deletes data inside the Email field', () => {
+      LoginPage.inputEmail.setValue(users.new.email);
+      LoginPage.clearInput(LoginPage.inputEmail)
+      expect(LoginPage.requiredMessage).toBeDisplayed();
+    });
+
+    it('TC-2.070 User deletes data inside the Email field and Password field', () => {
```
Comment (ayessetova) [inline test/specs/login.spec.js:61]: The test description is not clear. Did you mean that an error message appears if mandatory fields are empty?

**Review by ayessetova — CHANGES_REQUESTED** [review]

File: test/specs/roles/new.spec.js (hunk 1 above)
Comment (meyta2020) [reply test/specs/roles/new.spec.js:33]: fixed

File: test/specs/login.spec.js (hunk 2 above)
Comment (meyta2020) [reply test/specs/login.spec.js]: fixed

**Review by ayessetova — APPROVED** [review]

**Pull request merged**